src/main/java/dev/railroadide/logger/impl/DefaultLogger.java

```diff
@@ -207,17 +255,18 @@ private void beginWriteScheduling() {
      * Builder for creating a DefaultLogger instance.
      */
     public static class Builder {
+        //default builder attributes
         private final String name;
         private DateTimeFormatter loggingDateFormat = DateTimeFormatter.ofPattern("HH:mm:ss");
```

Comment:
This won't be needed anymore, so you might as well remove it and everywhere it's used.